### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Activities_Activitiestab_keyboard: keyboard focus goes illogical that after the Auto Resolve behavior button ,it goes to Cancel all button.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#1897]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS2.4.3; #Keyboard; #Win32;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
Windows10

**Repro Steps:**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Navigate to "Activities" Tab present on right bottom of the screen and hit enter.
6. Navigate to the list of items present under Activities Tab.
7. Navigate through tab key upto Cancel all and Auto Resolve behavior.
8. Check whether the keyboard focus goes illogical or not.
**​Actual:​**
While navigating through tab key,keyboard focus goes illogical that after the Auto Resolve behavior button ,it goes to Cancel all button.

**Expected:**
Keyboard focus should go to Cancel all button first and then go to Auto Resolve behavior.

**UserImpact:**
Keyboard user unable to get the content and control in logical way.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachment for Reference:**
[1897_A11y_AzureToolsStorageExplorer_Activities_Activitiestab_keyboard.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3646892/1897_A11y_AzureToolsStorageExplorer_Activities_Activitiestab_keyboard.pptx)

**Comment by MRayermannMSFT** [comment]
Jason merged a fix for this into master

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Verified on version 1.12.0 and the issue is no longer reproduced.Hence close this bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
